### PR TITLE
New ANTS pull request

### PR DESCRIPTION
**Pull request by TheChymera**
replacement for https://github.com/gentoo-science/sci/pull/435 which got all messed up on the git side of things...

### CODE REVIEW

File: sci-biology/ants/ants-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit cmake-utils git-r3
```
Comment (jlec) [inline sci-biology/ants/ants-9999.ebuild]: missing mulitlib eclass.

File: sci-biology/ants/ants-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+SLOT="0"
+LICENSE="BSD"
+KEYWORDS=""
+
+DEPEND="sci-libs/itk"
+RDEPEND="${DEPEND}"
+
+src_install() {
+	cd "${WORKDIR}/${P}_build/ANTS-build" || die "build dir not found"
+	emake DESTDIR="${D}" install
```
Comment (jlec) [inline sci-biology/ants/ants-9999.ebuild]: Exchange this by simply calling ```default```

Comment (TheChymera) [inline sci-biology/ants/ants-9999.ebuild]: so just replace line 23 by `default`?

Comment (jlec) [inline sci-biology/ants/ants-9999.ebuild]: yes

**Comment by TheChymera** [comment]
@jlec still no good?

**Pull request merged**